docs/event-bus/docs/03-02-event-flow-requirements.md

```diff
@@ -3,39 +3,29 @@ title: Event flow requirements
 type: Details
 ---
 
-The Event Bus enables a successful flow of the Events in Kyma when:
+The Event Bus enables a successful flow of Events in Kyma when:
 
-- The [EventActivation](#details-event-flow-requirements-activate-events) is in place.
-- You create a [Subscription](#details-event-flow-requirements-consume-events) Kubernetes custom resource and register the webhook for the lambda or a service to consume the Events.
+- You have enabled the [EventActivation](docs/components/event-bus#concepts) controller.
+- You have created a [Subscription](/docs/components/event-bus#custom-resource-subscription) custom resource and registered the webhook for a lambda or service to consume the Events.
 - The Events are [published](#details-event-flow-requirements-event-publishing).
 
-## Details
-
-See the following subsections for details on each requirement.
 
 ### Activate Events
 
-To receive Events, use EventActivation between the Namespace and the Application (App).
+Enable the EventActivation controller to ensure the Event flow between the Namespace and the Application (App). 
+Alternatively, use the [Console](/docs/components/console#overview-overview) to [bind](/docs/components/application-connector#getting-started-bind-an-application-to-a-namespace) the App to the Namespace and add the service to the Namespace using the Service Catalog.
 
-For example, if you define the lambda in the `test123` Namespace and want to receive the `order-created` Event type from the `ec-qa` App, you need to enable the EventActivation between the `test123` Namespace and the `ec-qa` App. Otherwise, the lambda cannot receive the `order-created` Event type.
+The diagram shows you the Event activation flow for a given Namespace.
 
-![EventActivation.png](./assets/event-activation.png)
+![EventActivation.png](./assets/event-activation.svg)
 
-### Consume Events
+The App sends the Events to the Event Bus and uses the EventApplication controller to ensure the Namespace receives the Events.  If you define a lambda in the `prod123` Namespace, it receives the **order.created** Event from the App through the Event Bus. The lambda in `test123` Namespace does not receive any Events, since you have not enabled the EventActivation controller.
```

Comment:
```suggestion
The App sends the Events to the Event Bus and uses the EventApplication controller to ensure the Namespace receives the Events.  If you define a lambda in the `prod123` Namespace, it receives the **order.created** Event from the App through the Event Bus. The lambda in the `test123` Namespace does not receive any Events, since you have not enabled the EventActivation controller.
```